### PR TITLE
added patient details in investigation report

### LINKED ISSUE #2280: Print Investigation summary
https://care.coronasafe.in/patient/76742faa-30dd-4eb9-9649-fe718f947ec0/investigation_reports

An investigation summary (in tabular format) may be generated for any patient. The print report on this page, lets the user print the table.

Add the following details on the print, on top of the table
- Name of the patient
- Age of the patient
- Name of hospital

### PR DESCRIPTION
**Pull request by Marmik2003**
Fixes #2280 
image

### CODE REVIEW

**Review by khavinshankar — APPROVED** [review]
LGTM

**Comment by nihal467** [comment]
test approved

File: src/Components/Facility/Investigations/Reports/ReportTable.tsx [hunk 1]
```diff
@@ -66,14 +69,15 @@ const ReportRow = ({ data, name, min, max }: any) => {
...
-      {data.map((d: any) => {
+      {data.map((d: any, index: number) => {
         const color = getColorIndex({
           min: d?.min,
           max: d?.max,
           value: d?.value,
         });
         return (
           <TableCell
+            key={index}
```
Comment (gigincg) [inline src/Components/Facility/Investigations/Reports/ReportTable.tsx]: Don't use the Array Index as the Key, unless there is no other unique id available.

File: src/Components/Facility/Investigations/Reports/ReportTable.tsx [hunk 2]
```diff
@@ -204,7 +226,8 @@ const ReportTable: React.FC<ReportTableProps> = ({
                       min={t.investigation_object.min_value}
                       max={t.investigation_object.max_value}
                       name={t.investigation_object.name}
-                      onChange={(e: { target: { value: any } }) => {}}
+                      // eslint-disable-next-line @typescript-eslint/no-empty-function
+                      onChange={() => {}}
```
Comment (gigincg) [inline src/Components/Facility/Investigations/Reports/ReportTable.tsx]: Can't this onChange prop be removed?

**Review by gigincg — CHANGES_REQUESTED** [review]
I don't think there is a need for a "Patient Details" Title. Simply having the Patient Name, Age etc below the "Report" Title should be enough

File: src/Components/Facility/Investigations/Reports/ReportTable.tsx (hunk 2 above)
Comment (Marmik2003) [reply src/Components/Facility/Investigations/Reports/ReportTable.tsx]: done

**Comment by gigincg** [comment]
@Marmik2003 The package-lock seems unrelated to the PR. Can you revert the package-lock or merge develop to clear the conflicts.

Also add a screenshot to the updated UI

**Comment by Marmik2003** [comment]
image

**Pull request merged**